testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -4828,7 +4828,7 @@ void testBitGetFunc(SqlOperatorFixture f, String functionName) {
           "2",
           "VARCHAR NOT NULL");
       f.checkString("to_char(timestamp '2022-06-03 13:15:48.678', 'IW')",
-          "23",
+          "22",
```

Comment:
this was a bug?

Reply:
Yes, same ISOWEEK issue, but for PostgreSQL:  https://github.com/apache/calcite/pull/3653#discussion_r1518608626